tests/acceptance/manager/ModelGroupsActionsCest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace hipanel\modules\stock\tests\acceptance\manager;
+
+use hipanel\helpers\Url;
+use hipanel\tests\_support\Page\IndexPage;
+use hipanel\tests\_support\Page\Widget\Input\Input;
+use hipanel\tests\_support\Step\Acceptance\Manager;
+
+/**
+ * @property string uni
+ */
+class ModelGroupsActionsCest
+{
+    /**
+     * @var IndexPage
+     */
+    private $index;
+    private $uni;
+
+    public function __construct()
+    {
+        $this->uni = uniqid();
+    }
+
+    public function _before(Manager $I)
+    {
+        $this->index = new IndexPage($I);
+    }
+
+    /**
+     * @param Manager $I
+     * @throws \Codeception\Exception\ModuleException
+     * @throws \Exception
+     */
+    public function ensureCreateModelGroupWorks(Manager $I): void
+    {
+        $I->needPage(Url::to('@model-group/create'));
+        $I->pressButton('Save');
+        $I->waitForPageUpdate();
+        $I->waitForText('Name cannot be blank.');
+        $I->click("//button[contains(@class, 'add-item')]");
+        $I->click("//button[contains(@class, 'add-item')]");
+        foreach (range(0,2) as $i) {
+            (new Input($I, "//input[@name='ModelGroup[$i][name]']"))
+                ->setValue("TEST_" . $this->uni . "_New_" . $i);
+            (new Input($I, "//textarea[contains(@name, 'ModelGroup[$i][descr]')]"))
+                ->setValue("Test description for $i item");
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_dtg]']"))
+                ->setValue(($i + 1) * 10);
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_sdg]']"))
+                ->setValue(($i + 1) * 10);
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_m3]']"))
+                ->setValue(($i + 1) * 10);
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_twr]']"))
+                ->setValue(($i + 1) * 10);
+        }
+        $I->pressButton('Save');
+        $I->waitForPageUpdate();
+        $I->closeNotification('Created');
+        $I->seeInCurrentUrl('/stock/model-group/index');
+
```

Comment:
```suggestion

        $I->amGoingTo('check that model groups I\'ve tried to create were indeed created');
```

В многоэтапных сценариях можно описывать свои намерения короткими аргументациям